packages/app/src/systems/DApp/pages/AddNetworkRequest/AddNetworkRequest.tsx

```diff
@@ -0,0 +1,75 @@
+import { cssObj } from '@fuel-ui/css';
+import { Button, Card, Text } from '@fuel-ui/react';
+
+import { useAddNetworkRequest } from '../../hooks';
+
+import { useAccounts } from '~/systems/Account';
+import { Layout, ConnectInfo } from '~/systems/Core';
+
+export function AddNetworkRequest() {
+  const { handlers, title, favIconUrl, origin, network } =
+    useAddNetworkRequest();
+  const { account } = useAccounts();
+
+  if (!origin || !network || !account) return null;
+
+  return (
+    <Layout title="Add Network Request" noBorder>
+      <Layout.Content css={styles.content} noBorder>
+        <ConnectInfo
+          origin={origin}
+          title={title || ''}
+          favIconUrl={favIconUrl}
+          headerText="Request to Add Network from:"
+        />
+        <Card css={styles.card} gap="$0">
+          <Card.Header space="compact">
+            Review the Network to be added:
+          </Card.Header>
+          <Card.Body css={styles.cardContentSection}>
+            <Text as="h2" fontSize="lg" color="intentsBase12">
+              {network.name}
+            </Text>
+            <Text fontSize="sm">{network.url}</Text>
+          </Card.Body>
+        </Card>
```

Comment:
this is looking different from the one in `Add Network` screen.

can we create a new component "ReviewNetworkCard" to make sure we use same styling and have more consistency ?